src/dashboard/components/Routes.test.tsx

```diff
@@ -21,7 +21,7 @@ const mockQueryResponse = [
     },
 ];
 
-const renderRoutesWithPath = (route: string = ''): RenderResult =>
+const renderRoutesWithPath = (route = ''): RenderResult =>
```

Comment:
I am a bit wary of removing this type :thinking: ... @Cooryd

Reply:
Oh yeah, good spot. I should have been more careful. I used the `--fix` flag to address all of the lint errors.

Reply:
This is the error:

```
$ yarn run lint
yarn run v1.19.1
$ eslint --ext .js,.ts,.tsx src/

/Users/nathanlisgo/libero/reviewer-client/src/dashboard/components/Routes.test.tsx
  24:31  error  Type string trivially inferred from a string literal, remove type annotation  @typescript-eslint/no-inferrable-types
```

Reply:
I've just checked an a compile error will be thrown if anything is assigned as a parameter other than a string when we just have `(route = '')` as the Type string is inferred.

```
ERROR in [at-loader] ./src/dashboard/components/Routes.test.tsx:37:57
    TS2345: Argument of type '123' is not assignable to parameter of type 'string'.
```

Reply:
This looks fine to me, I'm a bit confused by where the type inference is being set but lets go with it 👍

Reply:
@nlisgo / @Cooryd  .... interesting conversation about this here: https://stackoverflow.com/a/51048318